code/game/magic/Uristrunes.dm

```diff
@@ -147,36 +161,64 @@ var/list/uristrune_cache = list()
 					var/sw = I.GetPixel(x - 1, y - 1)
 
 					if(ne == "#000000" || se == "#000000" || nw == "#000000" || sw == "#000000")
-						I.DrawBox(rgb(200, 0, 0, 100), x, y)
+						I.DrawBox(bc2, x, y)
 
 	var/icon/result = icon(I, "")
 
-	result.Insert(I,  "", frame = 1, delay = 10)
+	I.MapColors(rgb(0x80,0,0,0), rgb(0,0x80,0,0), rgb(0,0,0x80,0), rgb(0,0,0,0xff))//we'll darken that color a bit
+
+	var/icon/I0 = icon(I, "")
+	I0.MapColors(rgb(0xff,0x08,0,0), rgb(0,0xff,0x08,0), rgb(0x08,0,0xff,0), rgb(0,0,0,0xff))
+	result.Insert(I0,  "", frame = 1, delay = 10)
 
 	if(animated == 1)
 		var/icon/I2 = icon(I, "")
-		I2.MapColors(rgb(0xff,0x0c,0,0), rgb(0,0,0,0), rgb(0,0,0,0), rgb(0,0,0,0xff))
-		I2.SetIntensity(1.04)
+		I2.MapColors(rgb(0xff,0x0c,0,0), rgb(0,0xff,0x0c,0), rgb(0x0c,0,0xff,0), rgb(0,0,0,0xff))
```

Comment:
Feel free to show me how you could reach the same result with matrices, because I read the DM documentation and got a headache out of trying to understand color matrices.